src/main/java/games/strategy/engine/config/PropertyFileReader.java

```diff
@@ -56,7 +57,7 @@ public String readProperty(final String propertyKey) {
       if (!props.containsKey(propertyKey)) {
         return "";
       } else {
-        return props.getProperty(propertyKey).trim();
+        return Strings.nullToEmpty(props.getProperty(propertyKey)).trim();
```

Comment:
props.getProperty is never null, because this is checked earlier.
Just remove the if-else clause around this.